thcrap/src/str_to_addr.asm

```diff
@@ -0,0 +1,137 @@
+/**
```

Comment:
For the other devs: this file was (in part) generated using the Clang compiler. Considering MSVC's problems, perhaps we could have a deeper discussion about switching the whole project over to Clang?